packages/client/components/ScopePhaseArea.tsx

```diff
@@ -71,33 +73,43 @@ const ScopePhaseArea = (props: Props) => {
   const {meeting} = props
   const [activeIdx, setActiveIdx] = useState(1)
   const isDesktop = useBreakpoint(Breakpoint.SIDEBAR_LEFT)
+  const {viewerMeetingMember} = meeting
+  if (!viewerMeetingMember) return null
+  const {user} = viewerMeetingMember
+  const {featureFlags} = user
+  const gitlabIntegration = viewerMeetingMember.teamMember.integrations?.gitlab
+  const hasGitLabProvider = !!(
```

Comment:
+1 _very_ nitty: `hasGitLabProvider` makes me think the viewer has already integrated (they "have" a gitlab integration).

Perhaps we say `isGitLabProviderAvailable`?